src/course-outline/utils/getErrorDetails.js

```diff
@@ -0,0 +1,24 @@
+import { API_ERROR_TYPES } from '../constants';
+
+export const getErrorDetails = (error, dismissible = true) => {
+  const errorInfo = { dismissible };
+  if (error.response?.status === 403) {
+    // For 403 status the error should be dismissible
```

Comment:
```suggestion
    // For 403 status the error should not be dismissible
```

Reply:
good catch :) fixed. Please merge it when the tests go green 😄